woocommerce-services.php

```diff
@@ -725,16 +731,27 @@ public function add_tracking_info_to_emails( $order, $sent_to_admin, $plain_text
 				return;
 			}
 
-			echo '<h2>' . __( 'Tracking', 'woocommerce-services' ) . '</h2>';
-			echo '<table class="td" cellspacing="0" cellpadding="6" style="margin-top: 10px; width: 100%; font-family: \'Helvetica Neue\', Helvetica, Roboto, Arial, sans-serif;">';
-			echo '<thead>';
-			echo '<tr>';
-			echo '<th class="td" scope="col">' . __( 'Provider', 'woocommerce-services' ) . '</th>';
-			echo '<th class="td" scope="col">' . __( 'Tracking number', 'woocommerce-services' ) . '</th>';
-			echo '</tr>';
-			echo '</thead>';
-			echo '<tbody>' . $markup . '</tbody>';
-			echo '</table>';
+			if ( $plain_text ) {
+				echo "\n=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n\n";
+				echo strtoupper( __( 'Tracking', 'woocommerce-services' ) ) . "\n\n";
```

Comment:
I'm not sure `strtoupper()` is "safe" for translated strings. (See: https://secure.php.net/manual/en/function.strtoupper.php#112383)

It might be better to just have the string be `'TRACKING'`, or perhaps make use of `mb_strtoupper()` with the `UTF-8` argument.